integration-tests/src/test/java/oracle/weblogic/kubernetes/ItLivenessProbeCustomization.java

```diff
@@ -231,6 +242,328 @@ void testCustomLivenessProbeNotTrigged() {
+    checkPodRestarted(domainUid, domainNamespace, adminServerPodName, adminPodCreationTime);
+
+    for (int i = 1; i <= NUMBER_OF_CLUSTERS_MIIDOMAIN; i++) {
+      for (int j = 1; j <= replicaCount; j++) {
+        String managedServerPodName =
+            domainUid + "-" + CLUSTER_NAME_PREFIX + i + "-" + MANAGED_SERVER_NAME_BASE + j;
+        checkPodRestarted(domainUid, domainNamespace, managedServerPodName,
+            managedServerPodsCreationTime.get(managedServerPodName));
+      }
+    }
+
+    // check the livenessProbe failureThreshold and successThreshold after the domain got patched
+    domain1 = assertDoesNotThrow(() -> getDomainCustomResource(domainUid, domainNamespace),
+        String.format("getDomainCustomResource failed with ApiException when tried to get domain %s in namespace %s",
+            domainUid, domainNamespace));
+    assertNotNull(domain1, "Got null domain resource");
+    assertNotNull(domain1.getSpec(), "domain1.getSpec() is null");
+    assertNotNull(domain1.getSpec().getServerPod(), "domain1.getSpec().getServerPod() is null");
+    assertNotNull(domain1.getSpec().getServerPod().getLivenessProbe(),
+        "domain1.getSpec().getServerPod().getLivenessProbe() is null");
+
+    // get the failureThreshold of livenessProbe after patch
+    failureThreshold = domain1.getSpec().getServerPod().getLivenessProbe().getFailureThreshold();
+    logger.info("livenessProbe failureThreshold after patch is: {0}", failureThreshold);
+    assertTrue(failureThreshold.intValue() == 3, "The livenessProbe failureThreshold after patch is not 3");
+
+    // get the successThreshold of livenessProbe after patch
+    successThreshold = domain1.getSpec().getServerPod().getLivenessProbe().getSuccessThreshold();
+    logger.info("livenessProbe successThreshold after patch is: {0}", successThreshold);
+    assertTrue(successThreshold.intValue() == 1, "The livenessProbe successThreshold after patch is not 1");
+
+    // verify the failureThreshold behavior of livenessProbe
+    // copy temp file to pod and verify the restartCount only happens after 1m 30 second
+    for (int i = 1; i <= NUMBER_OF_CLUSTERS_MIIDOMAIN; i++) {
+      for (int j = 1; j <= replicaCount; j++) {
+        String managedServerPodName =
+            domainUid + "-" + CLUSTER_NAME_PREFIX + i + "-" + MANAGED_SERVER_NAME_BASE + j;
+
+        // get the restart count of the container in pod before liveness probe restarts
+        int beforeRestartCount =
+            assertDoesNotThrow(() -> getContainerRestartCount(domainNamespace, null,
+                managedServerPodName, null),
+                String.format("Failed to get the restart count of the container from pod {0} in namespace {1}",
+                    managedServerPodName, domainNamespace));
+        logger.info("For server {0} restart count before liveness probe is: {1}",
+            managedServerPodName, beforeRestartCount);
+
+        String destLocation = "/u01/tempFile.txt";
```

Comment:
Instead of copying the file, maybe the test can run a command such as "touch /u01/tempFile.txt" or `echo "Test" > /u01/tempFile.txt" inside the pod to create the file.

Reply:
I just reused the method in the other tests.